data/mods/blazemod/blaze_blob_recipes.json

```diff
@@ -202,6 +202,18 @@
       [ [ "mutagen", 3 ], [ "iv_mutagen", 1 ] ]
     ]
   },
+  {
+    "result": "frostie_wheel",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "cooking",
+    "difficulty": 8,
+    "time": 15000,
+    "reversible": true,
```

Comment:
reversible?

Reply:
It's true for all parts that are craftable directly from a gelatinous/oozing/gray mass, so I added it to the gelacier wheel. I confess I don't know what I does, I was just trying to keep the style consistent. Should it get a second look?

Reply:
No, that's okay if similar other recipes are reversible.